.golangci.yml

```diff
@@ -0,0 +1,24 @@
+run:
+  timeout: 10m
+
+linters:
+  disable-all: true
+  enable:
+    - structcheck
```

Comment:
Prefer to add to https://github.com/clusternet/clusternet/blob/main/Makefile#L53